projects/bootloader/test/test_config.c

```diff
@@ -0,0 +1,171 @@
+#include "bootloader_mcu.h"
+#include "config.h"
+#include "crc32.h"
+#include "flash.h"
+#include "log.h"
+#include "persist.h"
+#include "soft_timer.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+extern StatusCode __real_persist_init(PersistStorage *persist, FlashPage page, void *blob,
+                                      size_t blob_size, bool overwrite);
+
+static PersistStorage s_test_persist_storage_1 = { 0 };
+static BootloaderConfig s_test_config_1 = { 0 };
+
+static PersistStorage s_test_persist_storage_2 = { 0 };
+static BootloaderConfig s_test_config_2 = { 0 };
+
+#define BOOTLOADER_CONFIG_PAGE_1_FLASH_PAGE (FLASH_ADDR_TO_PAGE(BOOTLOADER_CONFIG_PAGE_1_START))
+#define BOOTLOADER_CONFIG_PAGE_2_FLASH_PAGE (FLASH_ADDR_TO_PAGE(BOOTLOADER_CONFIG_PAGE_2_START))
+
+static void (*s_function_pointer_hook)() = NULL;
```

Comment:
The parameter list here (which is the `()` on the end) should be `(void)` instead of `()`.